### DISCUSSION TITLE
alert feature?

### DISCUSSION THREAD

**Discussion by acme64** [Rem0o/FanControl.Releases#1049] [Ideas]
Would it be possible to add some sort of alert feature? or maybe something as simple as changing the tray icon a different color? My water pumps is kind of flakey and sometimes it simply doesn't start up. If i can get fancontrol to warn me that'd be nice. As of right now sometimes i don't notice and the system will shutdown

**Comment by aidonaks** [comment]
Yes, an alert feature for say fan/pump failure would be extremely helpful.

**Comment by verbal666** [comment]
Also to me would be very very very helpful an alerting function, integrated or via plugin 👍
FC is a great piece of software, wonderful.

I used to install and use Gigabyte SIV on my Win, but, we know, GB tools are so poor and buggy!!! 🤦‍♂️... but SIV has an alerting function that works, also if although it often gave annoying false alarms 🤦‍♂️

Now i'm stable with FC, and it's really really great!!! 👏👏
